src/conventions/gmsh.jl

```diff
@@ -0,0 +1,103 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+abstract type GMSH <: Ordering end
+
+# Triangle
+# v
+# ^                 
+# |                 
+# 2                 
+# |`\               
+# |  `\             
+# |    `\           
+# |      `\         
+# |        `\       
+# 0----------1 --> u
+
+# counter-clockwise
+connectivity(::Type{<:Triangle{GMSH}}, ::Type{GMSH}, ::Val{1}) = connect.([(1, 2), (2, 3), (3, 1)], Segment)
+
+# Quadrangle
+#       v
+#       ^
+#       |
+# 3-----------2      
+# |     |     |      
+# |     |     |      
+# |     +---- | --> u
+# |           |      
+# |           |      
+# 0-----------1      
+
+# counter-clockwise
+connectivity(::Type{<:Quadrangle{GMSH}}, ::Type{GMSH}, ::Val{1}) = @SVector [(1, 2), (2, 3), (3, 4), (4, 1)]
+
+# Pyramid
+#                4
+#              ,/|\
+#            ,/ .'|\
+#          ,/   | | \
+#        ,/    .' | `.
+#      ,/      |  '.  \
+#    ,/       .' w |   \
+#  ,/         |  ^ |    \
+# 0----------.'--|-3    `.
+#  `\        |   |  `\    \
+#    `\     .'   +----`\ - \ -> v
+#      `\   |    `\     `\  \
+#        `\.'      `\     `\`
+#           1----------------2
+#                     `\
+#                        u
+
+function connectivity(::Type{<:Pyramid{GMSH}}, ::Type{GMSH}, ::Val{2})
+    base = connect((1,2,3,4), Quadrangle)
+    side = connect.([(1,2,5),(2,3,5),(3,4,5),(4,1,5)], Triangle)
+    [base; side]
+end
+
+
+# Tetrahedron
+#                    v
+#                  .
+#                ,/
+#               /
+#            2                 
+#          ,/|`\               
+#        ,/  |  `\             
+#      ,/    '.   `\           
+#    ,/       |     `\         
+#  ,/         |       `\       
+# 0-----------'.--------1 --> u
+#  `\.         |      ,/       
+#     `\.      |    ,/         
+#        `\.   '. ,/           
+#           `\. |/             
+#              `3              
+#                 `\.
+#                    ` w
+
+connectivity(::Type{<:Tetrahedron{GMSH}}, ::Type{GMSH}, ::Val{2}) = 
+    connect.([(1,2,3),(1,4,3),(1,4,2),(2,3,4)], Triangle)
+
```

Comment:
Sometimes you add 2 blank lines, sometimes you add 1 blank line between function definitions. I know this may sound picky, but it is important in the long run to be pedantic about these things.